xla/mlir/tools/mlir_interpreter/dialects/symbol_finder_windows.cc

```diff
@@ -0,0 +1,23 @@
+/* Implemented Function FindSymbolInProcess locate a function or variable
+ symbol in the current process's address space */
+
+#ifdef _WIN32
```

Comment:
Same here. Can you remove the ifdef?

Reply:
Fails on Linux on my end with an error
xla/mlir/tools/mlir_interpreter/dialects/symbol_finder_windows.cc:5:10: fatal error: windows.h: No such file or directory
    5 | #include <windows.h>
      |          ^~~~~~~~~~~
compilation terminated.